### DISCUSSION TITLE
How to merge the contents of multiple text boxes in the same cell?

### DISCUSSION THREAD

**Discussion by situchen** [jsvine/pdfplumber#430] [Ask for help with specific PDFs]
![image](https://user-images.githubusercontent.com/63528537/117639623-b09a5280-b1b6-11eb-9322-74bad6eceba6.png)
[1_test.pdf](https://github.com/jsvine/pdfplumber/files/6451105/1_test.pdf)

Desired result：
[[['变更前 ABC编号',
   '',
   '',
   '当前前ABC 编号',
   '',
   '',
   '测试人员',
   '',
   '测试单元',
   '',
   '电子子邮件',
   '',
   '联系电话',
   '',
   '备注',
   ''],
  ['',
   None,
   None,
   'ABC 000000000',
   None,
   None,
   '测试员A',
   None,
   None,
   '测试abc',
   None,
   'abcdefga@aaaaaa.com',
   None,
   '13188888888',
   None,
   None,
   '',
   None]]]

**Comment by situchen** [comment]
@jsvine

**Comment by samkit-jain** [comment]
In this PDF, when you run the debug table finder, you'll find that there are some hidden vertical lines as can be seen in the screenshot below which causes the cell to be split.
![image](https://user-images.githubusercontent.com/15127115/117764910-d4a77380-b24a-11eb-8727-2c08cff1818b.png)

To resolve the issue, you can filter out the hidden lines as mentioned in https://github.com/jsvine/pdfplumber/issues/285#issuecomment-710111180 which will give you the output as 
![image](https://user-images.githubusercontent.com/15127115/117765082-133d2e00-b24b-11eb-97a1-d2367b5f5ac2.png)

```
['变更前 ABC编号', '当前前ABC 编号', '测试人员', '测试单元', '电子子邮件', '联系电话', '备注']
['', 'ABC 000000000', '测试员A', '测试abc', 'abcdefga@aaaaaa.com', '13188888888', '']
```
